files/pod-template-file.yaml

```diff
@@ -47,15 +47,15 @@ spec:
   securityContext:
   {{- if contains "alpine"  .Values.defaultAirflowTag }}
     runAsUser: 100
-  {{- else }}
+  {{- else }}g
     runAsUser: 50000
   {{- end }}
   nodeSelector:
-    {{ toYaml .Values.nodeSelector | indent 8 }}
+{{ toYaml .Values.nodeSelector | indent 4 }}
```

Comment:
Something that will make this easier is the use of `nindent` makes a new line then indents the value, and does not depend at all on the spacing where `{{ toYaml` starts. For example.
```
nodeSelector: {{ toYaml .Values.nodeSelector | nindent 4 }}
```
Also arguably easier to read.